WWDC/NSTableView+Rx.swift

```diff
@@ -38,7 +38,7 @@ final class RxTableViewDelegateProxy: DelegateProxy, NSTableViewDelegate, Delega
     }
     
     func tableViewSelectionDidChange(_ notification: Notification) {
-        guard let numberOfRows = tableView?.numberOfRows, numberOfRows > 0 else { return }
```

Comment:
Required to allow the `SessionDetailViewController` to receive a blank state when a query results in no results.